ci-operator/step-registry/openshift-qe/installer/bm/ping/openshift-qe-installer-bm-ping-commands.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+set -x
+
+bastion=$(cat "/secret/hostname")
+
+ping -c 5 $bastion
```

Comment:
Looks like just `hostname` is being used to do a ping. Are we using ssh key as well?

Reply:
Why do we need to use the sshkey to ping?